website/static/js/logActionsList.json

```diff
@@ -1,4 +1,6 @@
 {
+    "created": "${user} created ${node}",
+    "deleted":  "${user} deleted ${node}",
```

Comment:
Not going to add all the necessary log actionshere since were are not yet exposing logs for preprints.